### ISSUE TITLE
About the hyperparameter setting.

### ISSUE THREAD

**Issue by YuQi9797** [bmartacho/UniPose#29]
Hello, sir.         
     I want to know the setting of learning rate. The code https://github.com/bmartacho/UniPose/blob/master/unipose.py#L52 , we can see  `self.step_size    = 13275`  ,  Why is the step size set to 13275 here? Or choose a larger number arbitrarily, are there any special requirements?
